### DISCUSSION TITLE
官方交流渠道

### DISCUSSION THREAD

**Discussion by Hk-Gosuto** [Hk-Gosuto/ChatGPT-Next-Web#283] [Announcements]
💡如果你在使用或者搭建过程中遇到了任何问题，可以使用下面两个渠道与我联系：

🔗 [Discord](https://discord.gg/QPrKZFwWn8)
🐧 QQ 群：`763467624`

**Comment by Silentely** [comment]
discord链接失效了

**Reply by Hk-Gosuto** [reply]
已更新
